study.md

```diff
@@ -54,5 +56,7 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+Streams and enumerables are both collections of data types.  However, elements in streams
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.